.changeset/tricky-socks-enjoy.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ux/axios-extension': patch
```

Comment:
An awesome new feature that deserves a minor version upgrade. :)

Reply:
Done :)